static/scripts/constants.ts

```diff
@@ -0,0 +1,30 @@
+import { txData } from "./render-transaction";
+
+export enum Chain {
+  Mainnet = "0x1",
+  Goerli = "0x5",
+  Gnosis = "0x64",
+}
+
+export enum Token {
+  DAI = "0x6b175474e89094c44da98b954eedeac495271d0f",
+  WXDAI = "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d",
+}
+
+export const chainName = {
+  [Chain.Mainnet]: "Ethereum Mainnet",
+  [Chain.Goerli]: "Goerli Testnet",
+  [Chain.Gnosis]: "Gnosis Chain",
+};
+
+export const chainExplorer = {
+  [Chain.Mainnet]: "https://etherscan.io",
+  [Chain.Goerli]: "https://goerli.etherscan.io",
+  [Chain.Gnosis]: "https://gnosisscan.io",
+};
+
+export const chainRpc = {
+  [Chain.Mainnet]: "https://rpc-pay.ubq.fi/v1/mainnet",
```

Comment:
I think this should be private. Perhaps this also can be fetched from chainlist etc. We can make a new bounty for this in order to close the current issue out faster.

Reply:
What do you mean by private?

Reply:
We should not share this RPC endpoint with the public because it can be abused (we don't have API keys etc.)

This originally was set up strictly for our internal tooling because we would hammer the RPC with a LOT of network requests (public RPC endpoints would rate limit us.)